### DISCUSSION TITLE
[BUG] ¿Cómo desactivar taskFilter creado por un usuario desactivado?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#94] [4. Bug Reports] [answered]
Hola quisiera saber lo siguiente:

En la vista task, en la sección de filtros, puede existir filtros de grupos creados por otros usuarios. Si el usuario A creó un filtro público A y posteriormente éste usuario se inactiva, dejando vigente el filtro público A para todos. Entonces la pregunta es: ¿Cómo lo podemos eliminar, ya que al revisar el flujo no se puede eliminar?

![image](https://user-images.githubusercontent.com/61419471/115262750-d1fda680-a102-11eb-9352-5469f74a23bb.png)

![image](https://user-images.githubusercontent.com/61419471/115263013-0ec99d80-a103-11eb-9b3d-6526b8ecf802.png)

Gracias

**Comment by acedward** [comment, marked as answer]
@NicolasPLG Esto al parecer es un bug.

**Reply by acedward** [reply]
Como workaround puedes activar el usuario y desactivar el task-filter global?

**Reply by NicolasPLG** [reply]
@acedward , no se puede el task global (por lo menos a mi no me funcionó).

Creo que sólo el usuario "createdBy" lo puede hacer.

Y si podemos reactivar el usuario o ingresar la cuenta (si es que está activo ya) y quitar la opción publico. La otra opción es por bd.

Lo levanto como un bug? o como un feature request? (usuario editor puede eliminar / quitar opcion filtro publico?

Me comentas :)

**Reply by acedward** [reply]
@NicolasPLG con esto el issue ya está levantado. 
Pero si necesitas urgencia por un cliente por favor habla con @jdurandpaz

**Reply by NicolasPLG** [reply]
@acedward perfecto si está levantado no hay problemas, seguirá el conducto regular.

Cierro la discusión :)
